runners/google-cloud-dataflow-java/src/test/java/org/apache/beam/runners/dataflow/DataflowRunnerTest.java

```diff
@@ -189,6 +209,30 @@ public SeekableByteChannel answer(InvocationOnMock invocation) throws Throwable
     when(mockGcsUtil.bucketAccessible(GcsPath.fromUri(VALID_PROFILE_BUCKET))).thenReturn(true);
```

Comment:
`
     when(mockGcsUtil.bucketAccessible(GcsPath.fromUri(VALID_STAGING_BUCKET))).thenReturn(true);
     when(mockGcsUtil.bucketAccessible(GcsPath.fromUri(VALID_STAGING_BUCKET))).thenReturn(true);
`
don't need the duplicate lines, last specified return is always returned on multiple calls.

Reply:
Didn't touch these lines but I'll dedupe.

Reply:
Dedupe disappeared?

Reply:
Hmm, signs point to rebase fail. Done again.